### DISCUSSION TITLE
Loading game error

### DISCUSSION THREAD

**Discussion by johnthesavage2** [voxeltycoon/issues#964] [Troubleshooting]
**Describe the bug*
Trying to load the game once the game begins to load the save game but then a notification pops up that says that it was unable to load because an object reference was not set to an instance

**Steps to reproduce**
Just loading any saved game 

**Build version**
0.87.1.6

**Platform**
Windows 10 Pro 21H1

**Reproducibility**
Every time I load a game

**Logs/saves**
![errormessage](https://user-images.githubusercontent.com/104240839/164865967-f6aace2b-47cf-4131-a708-2154c4316658.png)
[Player.log](https://github.com/voxeltycoon/issues/files/8545953/Player.log)

**Comment by andrewpey** [comment]
The problem comes from the [city growth boundary](https://steamcommunity.com/sharedfiles/filedetails/?id=2479765977) mod, you need to contact its author and see if they can fix it.
